torch_xla_py/xla_dist.py

```diff
@@ -6,9 +6,9 @@
 
 class Worker(object):
 
-  def __init__(self, internal_ip, mach_type, zone):
+  def __init__(self, internal_ip, machine_type, zone):
```

Comment:
We should have the options to disable the Client/Service machine uniform checks, not removing them.
By default we can enable them, but then we will feed in values coming from command line arguments, which will allow disabling them.

Reply:
Done. Also can you clarify what you mean by:

> but then we will feed in values coming from command line arguments, which will allow disabling them.

Like the VMs and TPUs when passed in through CLI? Or are you saying CLI should have a flag to disable checks?

Reply:
We will have two ways to specify clusters. One by GCP groups, and the other by command line.
The CLI should have options like `--allow_non_uniform_client_vm_set` and `--allow_non_uniform_service_vm_set`  (or similar), which we flow into the API.

Reply:
Correct. Thanks that sounds good to me.